blockchain/beacon/Cargo.toml

```diff
@@ -5,4 +5,26 @@ authors = ["ChainSafe Systems <info@chainsafe.io>"]
 edition = "2018"
 
 [dependencies]
-encoding = { package = "forest_encoding", path = "../../encoding" }
+grpc = { version = "0.7.1"}
+grpc-protobuf = { version = "0.7.1"}
+protobuf = "2.14.0"
+
+httpbis =  "0.8.0"
+tls-api = "0.3.2"
+tls-api-openssl = "0.3.2"
+crypto = { path = "../../crypto" }
+libsecp256k1 = "0.3.4"
+bls-signatures = "0.6.0"
+serde = { version = "1.0", features = ["derive"] }
+encoding = { path = "../../encoding", package = "forest_encoding" }
+sha2 = { version = "0.8", default-features = false }
+byteorder = "1.3.4"
+hex = "0.4.2"
+paired = "0.18.0"
+groupy = "0.3.0"
+
+[dev-dependencies]
+async-std = {version = "1.5.0", features = ["attributes"]}
+
+[build-dependencies]
+protoc-rust-grpc = { version = "0.7.1"}
```

Comment:
nit: can you remove the inconsistent spacing? (line after protobuf and inconsistent spaces in `{}`